### DISCUSSION TITLE
Complete Plugin Removal?

### DISCUSSION THREAD

**Discussion by Lorccan** [homebridge-simplisafe3/homebridge-simplisafe3#440] [Q&A] [answered]
I had the plugin set up and it seemed to be working fine. However, I then tried connecting homebridge to Node-RED using node-red-contrib-homebridge-automation. I could not get or set state of Simplisafe from Node-RED and then realised that SimpliSafe3 appeared twice in the list of accessories.

I tried to remove all trace of Simplisafe in homebridge by removing the plugin, removing the cached plugin data, and any other files containing the string SimpliSafe, and restarted homebridge and its server. When that didn't work, I completely removed Node-RED, reinstalled it, set up a new project, added the node again, and now (at least) only one "SimpliSafe3" was in the accessory list, but it's one too many.

Short of wiping out my extensive homebridge setup, is there a foolproof way of getting this back to zero, so that I can start over with a new installation of the plugin?

**Comment by shamoon** [comment]
I’m not really clear what happened but if you’ve already uninstalled the plugin you should be able to go in and remove any related accessories from homebridge directly under settings > accessories

**Reply by Lorccan** [reply]
Thanks. I have tried that but the reference is still there. I suppose I could try a plugin reinstall + re-auth to see if that overwrites the remaining reference.

**Reply by Lorccan** [reply, marked as answer]
I don't know what did it, but eventually things are how I would expect now.  Mysterious, but ultimately fixed!
